<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR_App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #134074;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>20</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; background-color: #eef4ed; padding: 5px; border-radius: 5px; text-align: center;</string>
    </property>
    <property name="text">
     <string>OCR System</string>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #eef4ed;</string>
    </property>
    <property name="text">
     <string>Extract text from images using OCR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8da9c4; color: #eef4ed; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Upload Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8da9c4; color: #eef4ed; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8da9c4; color: #eef4ed; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Back to Main</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
